internal/log/log.go

```diff
@@ -60,6 +60,19 @@ const (
 	spinnerText        = " waiting"
 )
 
+type SettingsInterface interface {
+	ApplySettings(tags string, skipOutput interface{})
+	SkipSuccess() bool
+	SkipFailure() bool
+	SkipSummary() bool
+	SkipMeta() bool
+	SkipExecution() bool
+	SkipExecutionOutput() bool
+	SkipExecutionInfo() bool
+	SkipSkips() bool
+	SkipEmptySummary() bool
```

Comment:
Since we move away from negation (`output` instead of `skip_output` and `Settings` instead of `SkipSettings`) I think it makes sense to change these methods to `LogSuccess(), LogFailure(), ...`. It sounds more natural, doesn't it?

WDYT?

Reply:
yes, I agree, but I kept the same methods for compatibility with the `SkipSettings` and to avoid changing `Skip*` methods everywhere in the `Lefthook` code and I did not want to change other places :)

If I change methods in `Settings`, it means I need to change methods in `SkipSettings` and change it in other places, but if it is ok if I rename `Skip*` methods everywhere, no problem I will do it! :)

Reply:
I think it makes sense to change the naming so it is more intuitive :+1:

Reply:
done!